core/chains/evm/log/broadcaster.go

```diff
@@ -494,12 +495,12 @@ func (b *broadcaster) onReplayRequest(replayReq replayRequest) {
 	b.backfillBlockNumber.Int64 = replayReq.fromBlock
 	b.backfillBlockNumber.Valid = true
 	if replayReq.forceBroadcast {
-		ctx, cancel := b.chStop.NewCtx()
+		ctx, cancel := b.chStop.CtxCancel(context.WithTimeout(context.Background(), time.Minute))
+		sqlutil.WithoutDefaultTimeout(ctx)
```

Comment:
Need to assign the result:
```suggestion
		ctx = sqlutil.WithoutDefaultTimeout(ctx)
```